lib/ical/stringify.js

```diff
@@ -209,16 +208,16 @@ stringify.property = function(property, designSet, noFold) {
  * If any of the above are present the result is wrapped
  * in double quotes.
  *
- * @function ICAL.stringify.paramPropertyValue
+ * @function ICAL.stringify.propertyParameterValue
  * @param {String} value      Raw property value
  * @param {boolean} force     If value should be escaped even when unnecessary
  * @return {String}           Given or escaped value when needed
  */
-stringify.paramPropertyValue = function(value, force) {
+stringify.propertyParameterValue = function(value, force) {
```

Comment:
This would break the API and we'd have to bump the major version to be true to semver. I'd suggest we avoid that for now.

Reply:
Noted this for later in #663